front/src/views/serials/serials.view.html

```diff
@@ -1,3 +1,27 @@
-<div>    
-  This is a serial page.
+<div class="Wrap">    
+  <el-row class="el-row-wrap" :gutter="20" type="flex" justify="start">
+    <el-col v-for="serial in getSerialsList" class="grid-col-wrap" :key = "serial.serialId">
+      <div class="grid-content-wrap">
+        <el-image fit='cover' :src="serial.posterImageSource | getImgSrc(baseUrl)"></el-image>
+        <div class="grid-content-main">
+          <div class="serial-name">
+            <div><span>{{serial.name}}</span></div>
+          </div>
+            <el-row type="flex" justify="start" class ="serial-rating">
```

Comment:
here you specify that it is serial rating not just rating

Reply:
you know, i am not sure what here comes but may be, it is useful to place common rate rules into common.css and specific for you view thing make in view styles